<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookDetails</class>
 <widget class="QWidget" name="BookDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:rgb(31, 13, 14);
background-color: rgb(85, 85, 127);
font-weight:700;
font-size:18px;
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>661</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="page_2">
    <widget class="QTableWidget" name="tableWidget_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>471</width>
       <height>301</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: rgb(85, 85, 127);
font-weight:700;
font-size:18px;
color: rgb(0, 0, 0)</string>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>100</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>250</number>
     </attribute>
     <column>
      <property name="text">
       <string>Book Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Count</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
font: 700 16pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>All books</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_5">
    <widget class="QTableWidget" name="tableWidget_7">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>471</width>
       <height>301</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: rgb(85, 85, 127);
font-weight:700;
font-size:18px;
color: rgb(0, 0, 0)</string>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>100</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>250</number>
     </attribute>
     <column>
      <property name="text">
       <string>Book Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Count</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
font: 700 16pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Non-Technical</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page">
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>471</width>
       <height>301</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: rgb(85, 85, 127);
font-weight:700;
font-size:18px;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>100</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>250</number>
     </attribute>
     <column>
      <property name="text">
       <string>Book Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Count</string>
      </property>
     </column>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>661</width>
       <height>451</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page_3">
      <widget class="QTableWidget" name="tableWidget_3">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>40</y>
         <width>471</width>
         <height>301</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color: rgb(85, 85, 127);
font-weight:700;
font-size:18px;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>100</number>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>250</number>
       </attribute>
       <column>
        <property name="text">
         <string>Book Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Count</string>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QWidget" name="page_4">
      <widget class="QTableWidget" name="tableWidget_4">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>100</y>
         <width>471</width>
         <height>301</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color: rgb(85, 85, 127);
font-weight:700;
font-size:18px;
color: rgb(0, 0, 0)</string>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>100</number>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>250</number>
       </attribute>
       <column>
        <property name="text">
         <string>Book Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Count</string>
        </property>
       </column>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>40</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
font: 700 16pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Technical Books</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>540</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>All</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>540</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Non-Technical</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>540</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Technical</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
